esp8266-wordclock.ino

```diff
@@ -90,10 +92,10 @@ byte twaalf_uur[11] = {110, 91, 90, 71, 70, 51, 0, 0, 0, 0, 0};
 byte* hours_mask[12] = {een_uur, twee_uur, drie_uur, vier_uur, vijf_uur, zes_uur, zeven_uur, acht_uur, negen_uur, tien_uur, elf_uur, twaalf_uur};
 
 // Single minutes
-byte min_1[11] = {113,   0,   0,  0, 0, 0, 0, 0, 0, 0, 0};
-byte min_2[11] = {113, 112,   0,  0, 0, 0, 0, 0, 0, 0, 0};
-byte min_3[11] = {113, 112, 111,  0, 0, 0, 0, 0, 0, 0, 0};
-byte min_4[11] = {113, 112, 111, 114, 0, 0, 0, 0, 0, 0, 0};
+byte min_1[11] = {112,   0,   0,  0, 0, 0, 0, 0, 0, 0, 0};

```

Comment:
Dit moeten we even in een aparte file doen en ignoren op git denk ik

Reply:
Niet ignoren want is wel cruciaal maar wel losse file denk ik idd

Reply:
Ooh je hebt andere nummering, Ik weet niet hoe mijn nummer worden.

Reply:
Ik zie nu dat mijn nummering hetzelfde moet zijn als die bij jou. Dus dit kunnen we gewoon hardcoden 👍